extendeddatacrossword.go

```diff
@@ -274,9 +274,16 @@ func (eds *ExtendedDataSquare) rebuildShares(
 func (eds *ExtendedDataSquare) verifyAgainstRowRoots(
 	rowRoots [][]byte,
 	r uint,
-	shares [][]byte,
+	oldShares [][]byte,
+	rebuiltIndex int,
+	rebuiltShare []byte,
 ) error {
-	root := eds.computeSharesRoot(shares, Row, r)
+	var root []byte
+	if rebuiltIndex < 0 || rebuiltShare == nil {
```

Comment:
Question: Isn't `rebuiltShare == nil` sufficient here to indicate this branch? Or can it happen in other cases?

Reply:
Ah, nvm.